docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md

```diff
@@ -79,7 +77,7 @@ Check the resulting zip file's size before you proceed. The zipped bundle should
 
 ### Bundle and push
 
-The `stencil push` command is available only for themes that you have successfully initialized using an OAuth token (with `Themes: modify scope`). This command bundles your theme and uploads it to the associated store, in one continuous process.
+The `stencil push` command is available only for themes that you have successfully initialized using an [Stencil CLI token](https://support.bigcommerce.com/s/article/Store-API-Accounts#creating) (with `Themes: modify scope`). This command bundles your theme and uploads it to the associated store, in one continuous process.

```

Comment:
Given we have a "Stencil CLI token" option in the UI now, I think this is a better way to describe the process.